automated-comments/go/general/use_variable_assignment.md

```diff
@@ -0,0 +1 @@
+You are declaring variable `%{name}` only to assign a value to it afterwards. You can declare the variable by assigning a value to it in one step.
```

Comment:
Same here with mentioning the variable this is about. Suggestion:

> Rather than declaring the variable `%{name}` first, and then assigning a value to it afterwards, it is more common to declare and assign in a single statement.

Reply:
Note: We are in `general` section here which applies to all exercises. In later exercises it might not be obvious which variable this refers to otherwise.

Reply:
@tehsphinx Yes - interpolating is good.